result/block-scoped-var.js

```diff
@@ -0,0 +1,57 @@
+/*eslint block-scoped-var: "error"*/
+
+function doIf() {
+    if (true) {
+        var build = true;
+    }
+
+    console.log(build);
+}
+
+function doIfElse() {
+    if (true) {
+        var build = true;
```

Comment:
__[development][eslint]__
* Error - 'build' used outside of binding context. (block-scoped-var)
* Error - 'build' is defined but never used (no-unused-vars)